docs/essentials/controls.md

```diff
@@ -0,0 +1,208 @@
+---
+title: 'Controls'
+---
+
+<div style="background-color:#F8FAFC">
+TODO add same gif used in the SB 6.0 docs
+</div>
+
+Storybook Controls gives you a graphical UI to interact with a component's arguments dynamically, without needing to code. It creates an addon panel next to your component examples ("stories"), so you can edit them live.
+
+It does not require any modification to your components, and stories for controls are:
+
+- Convenient. Auto-generate controls based on React/Vue/Angular/etc. components.
+- Portable. Reuse your interactive stories in documentation, tests, and even in designs.
+- Rich. Customize the controls and interactive data to suit your exact needs.
+
+To use the controls addon, you need to write your stories using [args](../writing-stories/args). Storybook will automatically generate controls based on your args and what it can infer about your component; but you can configure the controls further using [argTypes](../api/stories#argtypes), see below.
+
+<div style="background-color:#F8FAFC">
+ask tom/dom/michael for the link of the migration guide.
```

Comment:
@shilman?

Reply:
Link will be https://medium.com/storybookjs/storybook-6-migration-guide-200346241bb5

Reply:
Link added. Marking this as solved.